Pinta.Tools/Tools/GradientTool.cs

```diff
@@ -30,7 +30,7 @@
 
 namespace Pinta.Tools
 {
-	public class GradientTool : BaseTool
+	public sealed class GradientTool : BaseTool
```

Comment:
This produced some `new protected member declared in sealed type` warnings

Reply:
I just made the field `private`. If the class is ever unsealed, whether or not the members are made `protected` will have to be decided conscientiously and on a case-by-case basis